src/esn.calendar.libs/app/services/fc-moment.js

```diff
@@ -1,5 +1,11 @@
 const moment = require('moment');
 
+require('moment/locale/fr');
+require('moment/locale/en-gb');
+require('moment/locale/vi');
+require('moment/locale/ru');
+require('moment/locale/zh-cn');
+
```

Comment:
1. Isn't English supported by default?
2. Importing the locales directly like this may be a little bit fragile: When we add a new locale, we may forget to require the `moment`'s locales here, and end up having some bugs with the new locale. Even if in the future we know that the bugs exist, we may not know that the missing locale should be added here (no single source of truth). You may need to figure out a better way to do this for our frontend, but in the ESN backend, I did this:

    - https://ci.linagora.com/linagora/lgs/openpaas/esn/merge_requests/1309
    - https://ci.linagora.com/linagora/lgs/openpaas/linagora.esn.calendar/merge_requests/674/diffs#ade79c4f8d1ab9f4b98ca2f9c1139440d5764742_0_14

Reply:
created an issue for this:
https://github.com/OpenPaaS-Suite/esn-frontend-calendar/issues/160